rocky/tools/forms/ooi_form.py

```diff
@@ -75,6 +75,7 @@ def generate_form_fields(self, hidden_ooi_fields: dict[str, str] | None = None)
                 or annotation == dict[str, str]
                 or annotation == list[str]
                 or annotation == dict[str, JsonValue]
+                or annotation == dict[str, Any]
```

Comment:
Adding this line makes the conditional checks for `dict[str, str]` and `dict[str, JsonValue]` redundant. I suggest removing these checks